grpc.proto

```diff
@@ -28,6 +28,12 @@ message RecoverResponse {
     Response response = 1; 
 }
 
+// Keygen's success response
+message KeygenOutput {
+    bytes pub_key = 1;               // pub_key; common for all parties
+    bytes group_info = 2;            // recovery info of all parties' shares; common for all parties
+    repeated bytes recovery_info= 3; // private recovery info of this party's shares; unique for each party
```

Comment:
why is `recovery_info` of type `repeated bytes` whereas `group_info` has type `bytes`?

Reply:
`recovery_info` is a vector of `BytesVec`, ie `Vec<Vec<u8>>`. https://github.com/axelarnetwork/tofn/blob/main/src/gg20/keygen/secret_key_share.rs#L160
`group_info` is a serialized blob of bytes from tofnd's perspective, so it's just an array of bytes. https://github.com/axelarnetwork/tofn/blob/main/src/gg20/keygen/secret_key_share.rs#L79

Reply:
Is there a reason why `recovery_info` is a `Vec<BytesVec>`?  @milapsheth should we change tofn so that it's a serialized blob like `group_info`?

Reply:
The only reason this is necessary is if axelar-core performs some kind of validation depending on the length of `recovery_info` - tofnd does not.

Reply:
After discussion, we decided to convert `recovery_info` to `bytes`. `tofnd` will manage the serialization.